composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.7",
+        "ezsystems/ezpublish-kernel": "^6.7@dev",
```

Comment:
This is ok, we do the same in solr: https://github.com/ezsystems/ezplatform-solr-search-engine/blob/master/composer.json#L15